frontend/src/queries/nodes/HogQLX/render.tsx

```diff
@@ -44,7 +45,6 @@ export function renderHogQLX(value: any): JSX.Element {
             )
         } else if (tag === 'RecordingButton') {
             const { sessionId, ...props } = rest
```

Comment:
Worth adding https://www.npmjs.com/package/eslint-plugin-react-hooks to avoid in the future?